src/ppx_irmin/bin/ppx_irmin.ml

```diff
@@ -55,6 +59,6 @@ let () =
     (Arg.String (function "" -> library := None | s -> library := Some s))
     ~doc;
 
-  Deriving.add ~str_type_decl:str_type_decl_generator
-    ~sig_type_decl:sig_type_decl_generator ppx_name
-  |> Deriving.ignore
+  Reserved_namespaces.reserve ppx_name;
```

Comment:
Why do we need to do this?

Reply:
We don't _need_ this, strictly speaking, but it's good hygiene to ensure that we never end up in a situation where multiple rewriters try to define functionality for the same attributes. Any other PPXes that tried to declare a `@@irmin` attribute would fail when run with this PPX.

Reply:
I see, thanks!